server/prometheus.js

```diff
@@ -8,6 +8,17 @@ const commonLabels = [
     'monitor_port',
 ]
 
+const monitor_cert_days_remaining = new PrometheusClient.Gauge({
+    name: 'monitor_cert_days_remaining',
+    help: 'The number of days remaining until the certificate expires',
+    labelNames: commonLabels
+});
+
+const monitor_cert_is_valid = new PrometheusClient.Gauge({
```

Comment:
Is there need for a dedicated gauge for this, given it's very simple for the downstream to just check whether `monitor_cert_days_remaining == 0`?

Reply:
Possibly not, I'd rather monitor it directly and have too many metrics than rely on "aggregation", but happy to leave it out if that's the consensus...